topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: tutorial_hands_on
+
+title: Peptide Library Data Analysis
+level: Intermediate
+questions:
+  - How to utilize quantitative properties of amino acids and peptide sequence to analyse peptide data?
+objectives:
+  - Calculate descriptors
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
```

Comment:
It would be helpful to make the motivation and aim of this GTN clearer. A paragraph similar to “background” and “calculating peptide descriptors” in the ML-modeling training would be great.

Reply:
The introduction has been rewritten, which covers these points.

Reply:
added with the introduction